default_config.edn

```diff
@@ -20,23 +20,27 @@
   :international-out-migrants
   "./datasets/default_datasets/migration/historic_migration_flows_international_out.csv"
   :future-mortality-trend-assumption
-  "./datasets/default_datasets/mortality/future_mortality_trend_assumption.csv"}
+  "./datasets/default_datasets/mortality/future_mortality_trend_assumption.csv"
+  :future-fertility-trend-assumption
+  "./datasets/default_datasets/fertility/future_fertility_trend_assumption.csv"}
 
  :user-parameters
  {;; Core module
   :first-proj-year 2015
   :last-proj-year 2040
 
   ;; Fertility module
+  :fert-variant :fixed
```

Comment:
@sunnytownsend said in an earlier branch that the defaults should be the apply national trend methods, and this is now the case outside the demo tool. For consistency, change :fert-variant to :applynationaltrend